plugins.json

```diff
@@ -10513,6 +10513,143 @@
     },
     "updated_at": "2021-10-06T05:48:52.017186294Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-zoom",
+    "icon_data": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iOTEyIiBoZWlnaHQ9IjkxMiIgdmlld0JveD0iMCAwIDkxMiA5MTIiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxjaXJjbGUgY3g9IjQ1NiIgY3k9IjQ1NiIgcj0iNDU2IiBmaWxsPSIjMkQ4Q0ZGIi8+CjxwYXRoIGQ9Ik0xOTkuNSAzNDkuMTI1QzE5OS41IDMzMy4zODUgMjEyLjI2IDMyMC42MjUgMjI4IDMyMC42MjVINDcwLjI1QzUyOS4yNzUgMzIwLjYyNSA1NzcuMTI1IDM2OC40NzUgNTc3LjEyNSA0MjcuNVY1NjIuODc1QzU3Ny4xMjUgNTc4LjYxNSA1NjQuMzY1IDU5MS4zNzUgNTQ4LjYyNSA1OTEuMzc1SDMwNi4zNzVDMjQ3LjM1IDU5MS4zNzUgMTk5LjUgNTQzLjUyNSAxOTkuNSA0ODQuNVYzNDkuMTI1WiIgZmlsbD0id2hpdGUiLz4KPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik02NzYuODE1IDU4My44MjlMNjE4LjI5MyA1MzguNjUzQzYxNC43OTcgNTM1Ljk1NSA2MTIuNzUgNTMxLjc4OSA2MTIuNzUgNTI3LjM3M1Y0MTMuMTI3QzYxMi43NSA0MDguNzExIDYxNC43OTcgNDA0LjU0NSA2MTguMjkzIDQwMS44NDdMNjc2LjgxNSAzNTYuNjcxQzY4MS4zNDUgMzUyLjAxNiA2ODcuNjc5IDM0OS4xMjUgNjk0LjY4OCAzNDkuMTI1QzcwOC40NiAzNDkuMTI1IDcxOS42MjUgMzYwLjI5IDcxOS42MjUgMzc0LjA2MlY1NjYuNDM4QzcxOS42MjUgNTgwLjIxIDcwOC40NiA1OTEuMzc1IDY5NC42ODggNTkxLjM3NUM2ODcuNjc5IDU5MS4zNzUgNjgxLjM0NSA1ODguNDg0IDY3Ni44MTUgNTgzLjgyOVoiIGZpbGw9IndoaXRlIi8+Cjwvc3ZnPgo=",
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-zoom-v1.6.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.6.0",
+    "hosting": "on-prem",
+    "author_type": "mattermost",
+    "release_stage": "production",
+    "enterprise": false,
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmKduCgACgkQ0bVLR6XO/sQMlRAAgGIaoNG0E6ZDFbhscqNeviFQhZPZDcDszYyeQ8heMBwlMVuq/6nchmaG5xoZIUQLmOPBeeSDr67Et4WvrJy7FsC84RxZdYynQoRpuStVfoj/pzyc2LK857GgH+XmL5fY7RGGmgCI9ZT+Yso4hdO+7dv4qpc4hIR9ZcOwBPtlPVpLpRqlRT5abUFEnzQZUduYD6uncSDedRkjCOaWHmop2+Ayk21qh3MQDXqpvBkGFskEpXYCsj5NEr30pSGWr+tSRWh4DpbrGhwHPkhVGyDaTMXw56QGi9P8hZ3NuMvkyvWTQmCcTOLDKUK8+nnW8a6sCvcvIuOtMqnUkMkuSk5r5gnOoh1eB8zOTgrdNUkNcrMFgkiC/xusRq3qBAXGVsywDL6mjSpBwruKwQ5F6b8w8Ftl/x0V5NktqqixpobR5lm11vCh7Scu5AGoSH3jFHA5VyP3lYGxY6hr6OQMC+0VHpZBfqLIJtG6AQcSutmMN5RLnov6Ksbz1c3mV7iXMbaOocp1pU8ZVbo/3MJjzCzz/9ChJ0kFD/KPEdJJiJDJ8Jlw9zvxfRt0tNt3fAFZj9+LgFh2dncdxoS2a7O2uFwdTLP2JA/HoqW0H/JrrEW6/QgMjaVp7ygMI2SeehE7pEzXWI5n/dKMLAktw1O5Uzk0hOhW+O5LRk9ZIsBmv9Hv8KA=",
+    "repo_name": "mattermost-plugin-zoom",
+    "manifest": {
+      "id": "zoom",
+      "name": "Zoom",
+      "description": "Zoom audio and video conferencing plugin for Mattermost 5.2+.",
```

Comment:
Not blocking now, but @mickmister we should probably update this description at some point, since a few lines lower it says "min_server_version: 5.12.0".